cli/tests/setup_suite.bash

```diff
@@ -474,6 +480,11 @@ common_suite_setup() {
   } >&3
 }
 
+populate_scrape_cache() {
+  "$PKGDB_BIN" scrape "$PKGDB_NIXPKGS_REF_NEW" legacyPackages
+  "$PKGDB_BIN" scrape "$PKGDB_NIXPKGS_REF_OLD" legacyPackages
```

Comment:
question blocking: can we back this out now? Will make running tests locally sometimes slower

Reply:
yeah didnt really change things
The issue seems to arise after initializing but its not quite worth to spend too much time investigating further

Reply:
removed